source/features/repo-age.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import timeAgo from '../libs/time-ago';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import * as icons from '../libs/icons';
+import {getRepoGQL} from '../libs/utils';
+
+async function getRepoCreationDate(): Promise<Date> {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			createdAt
+		}
+	`);
+
+	return new Date(repository.createdAt);
```

Comment:
@maxmclau could you add long-term caching to this value?
Something like

```js
const key = __featureName__ + ':' + getRepoURL();
if (await cache.has(key)) {
	return cache.get(key);
}

const {repository} = await api.v4(`
	repository(${getRepoGQL()}) {
		createdAt
	}
`);

cache.set(key, repository.createdAt);
return repository.createdAt;
```

Reply:
#2578 merged

Reply:
@fregante this memoization implementation is pretty neat - I opened a PR that should close this.